Formula/klaytn.rb

```diff
@@ -0,0 +1,55 @@
+class Klaytn < Formula
+  desc "Klaytn Blockchain Platform"
+  homepage "https://docs.klaytn.com"
```

Comment:
Actually, Klaytn's homepage is https://www.klaytn.com :)

Reply:
I think https://www.klaytn.com is for the company Klaytn, not for the platform Klaytn.

Reply:
OK. Let's use https://docs.klaytn.com for now.